Library/Homebrew/cask/cmd/install.rb

```diff
@@ -56,7 +56,8 @@ def self.install_casks(
         require_sha: nil,
         quarantine: nil,
         quiet: nil,
-        zap: nil
+        zap: nil,
+        dry_run: nil
```

Comment:
```suggestion
        dry_run: nil,
```

unless it fails `brew style`